docs/output.md

```diff
@@ -1,63 +1,28 @@
 # nf-core/scflow: Output
 
-## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/scflow/output](https://nf-co.re/scflow/output)
-
-> _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
-
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline.
 
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
 <!-- TODO nf-core: Write this documentation describing your workflow's output -->
```

Comment:
```suggestion
```

Reply:
@nfancy

Reply:
An updated output.md is in the works

Reply:
An updated output.md is present in the latest commit 079018b1c9ab5a4cec4824d9ab0ff5bde77dffc8